core/src/context.rs

```diff
@@ -286,7 +282,7 @@ impl Context {
                 .agent()
                 .chain_store()
                 .iter_type(&Some(top), &EntryType::CapTokenGrant)
-                .nth(0)?
+                .next()?
```

Comment:
What is the difference between `next()` and `nth(0)`? Why did you change this? Does `nth(0)` break anything wrt signature verification?